database/table_creation/table_teams.sql

```diff
@@ -1,18 +1,21 @@
 ATTACH DATABASE '/Users/aliemre2023/Downloads/archive (8)/nba.sqlite' AS nba_original;
+
 CREATE TABLE IF NOT EXISTS teams (
     team_id INTEGER PRIMARY KEY,
-    abbreviation VARCHAR NOT NULL,
-    nickname VARCHAR NOT NULL,
-    year_founded TIMESTAMP,
+    abbreviation VARCHAR NOT NULL CHECK (length(abbreviation) = 3), -- Ensure abbreviation is 3 characters long
+    nickname VARCHAR NOT NULL CHECK (length(trim(nickname)) > 0), -- Ensure nickname is not empty or only spaces
+    name TEXT NOT NULL DEFAULT '' CHECK (length(trim(name)) > 0), -- Ensure name is not empty or only spaces
+    year_founded TIMESTAMP CHECK (year_founded > '1900-01-01'), -- Ensure year founded is after 1900
     city_id INTEGER,
     arena_id INTEGER,
     owner TEXT,
-    general_manager TEXT,
+    general_manager TEXT, 
     headcoach TEXT,
     dleague_affiliation TEXT,
     facebook TEXT,
     instagram TEXT,
     twitter TEXT,
+    logo_url TEXT,
```

Comment:
if you decided to add name and logo_url column there, also consider the rest of the sql file, because I can't run this file in my computer.

Reply:
also at the end of the file the logo_url is redefined, sql does not allow me to do this (there are 2 columns with the same name)